pkg/client/group_mgmt_client.go

```diff
@@ -393,7 +393,7 @@ func processAsyncResponse(client *GroupMgmtClient, body []byte) (interface{}, er
 			}
 		}
 		if len(jobId) == 0 {
-			return nil, fmt.Errorf("http response error: status (202), failed to get the job id")
+			return nil, fmt.Errorf("http response error: status (202), failed to get the job id, message: %v",errResp)
```

Comment:
I suggest changing "failed to get the job id" to "but no job ID returned".  Also change "message" to "messages" for consistency with the other errors.

Reply:
sure